R/metrics-sample.R

```diff
@@ -215,7 +215,25 @@ dss_sample <- function(observed, predicted, ...) {
 #' function from the
 #' \pkg{scoringRules} package. Can be used for continuous as well as integer
 #' valued forecasts
+#'
+#' The Continuous ranked probability score (CRPS) can be interpreted as the sum
```

Comment:
Add a reference for this approach. In particular where does it sit within the methods discussed here https://arxiv.org/pdf/2311.14122

Reply:
Thanks for opening the issue!